script/verification/SystemConfigUpgradeEcotoneScalars.s.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {console2 as console} from "forge-std/console2.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {SuperchainRegistry} from "script/verification/Verification.s.sol";
+import "@eth-optimism-bedrock/src/dispute/lib/Types.sol";
+import {ISystemConfig} from "./ISystemConfig.sol";
+import {IResourceMetering} from "./IResourceMetering.sol";
+import {MIPS} from "@eth-optimism-bedrock/src/cannon/MIPS.sol";
+import {Simulation} from "@base-contracts/script/universal/Simulation.sol";
+import {NestedMultisigBuilder} from "@base-contracts/script/universal/NestedMultisigBuilder.sol";
+import {SystemConfigUpgrade} from "script/verification/SystemConfigUpgrade.s.sol";
+
+contract SystemConfigUpgradeEcotoneScalars is SystemConfigUpgrade {
+    using LibString for string;
+
+    constructor(string memory l1ChainName, string memory l2ChainName, string memory release)
+        SystemConfigUpgrade(l1ChainName, l2ChainName, release)
+    {}
+
+    /// @notice Public function that must be called by the verification script.
+    function checkSystemConfigUpgrade() public view override {
+        ISystemConfig sysCfg = ISystemConfig(proxies.SystemConfig);
+        uint256 reencodedScalar =
+            (uint256(0x01) << 248) | (uint256(sysCfg.blobbasefeeScalar()) << 32) | sysCfg.basefeeScalar();
+        console.log(
+            "checking baseFeeScalar and blobbaseFeeScalar ",
+            LibString.toString(sysCfg.basefeeScalar()),
+            LibString.toString(sysCfg.blobbasefeeScalar())
+        );
+        if (
+            uint8(previous.scalar >> 248) == 1 // most significant bit
+        ) {
```

Comment:
Why is this behind an `if` block? A code comment for what this method is testing would be helpful for people not familiar with this encoding

Reply:
Tidied that up here [5253651](https://github.com/ethereum-optimism/superchain-ops/pull/418/commits/5253651eb9ce4c764cbc22e6cd32c0c3a26d01eb) PTAL hopefully it is clear now.